src/Resources/public/Admin.js

```diff
@@ -126,6 +126,7 @@ var Admin = {
                     theme: 'bootstrap',
                     dropdownAutoWidth: true,
                     minimumResultsForSearch: minimumResultsForSearch,
+                    placeholder: allowClearEnabled ? ' ' : '', // allowClear needs placeholder to work properly
```

Comment:
The same solution is applied here: https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig#L72